codec.proto

```diff
@@ -9,21 +9,19 @@ import "google/protobuf/descriptor.proto";
 
 // used in Vote
 enum SignedMsgType {
-  option (gogoproto.goproto_enum_stringer)  = true;
-  option (gogoproto.goproto_enum_prefix)    = false;
+  option(gogoproto.goproto_enum_stringer) = true; option(gogoproto.goproto_enum_prefix) = false;
 
-  SIGNED_MSG_TYPE_UNKNOWN   = 0 [(gogoproto.enumvalue_customname) = "UnknownType"];
+  SIGNED_MSG_TYPE_UNKNOWN = 0 [(gogoproto.enumvalue_customname) = "UnknownType"];
   // Votes
   SIGNED_MSG_TYPE_PREVOTE   = 1 [(gogoproto.enumvalue_customname) = "PrevoteType"];
   SIGNED_MSG_TYPE_PRECOMMIT = 2 [(gogoproto.enumvalue_customname) = "PrecommitType"];
   // Proposals
-  SIGNED_MSG_TYPE_PROPOSAL  = 32 [(gogoproto.enumvalue_customname) = "ProposalType"];
+  SIGNED_MSG_TYPE_PROPOSAL = 32 [(gogoproto.enumvalue_customname) = "ProposalType"];
```

Comment:
I think the formatting broke here because of the comments

Reply:
I'll revert formatting from this file, shouldn't touch it

Reply:
I had to update base branch cosmos, now it looks better